src/workflows/import-docmap.ts

```diff
@@ -19,9 +19,9 @@ export async function importDocmap(url: string): Promise<DocMapImportOutput> {
   const result = await parseDocMap(docMap);
 
   await Promise.all(
-    result.versions.map(async (version) => executeChild('importContent', {
+    result.versions.map(async (version, index) => executeChild('importContent', {
       args: [version],
-      workflowId: 'import-content',
+      workflowId: `import-content-${version.doi}-${index}`,
```

Comment:
I think it would be sufficient to use index. Certainly the doi is problematic because of range of characters that it supports.

Reply:
For some context: https://github.com/elifesciences/enhanced-preprints-issues/issues/582

I'm happy to approve if we can remove the doi from the workflowId.